backend/api/student/simulation_instances.py

```
@@ -185,28 +185,24 @@ async def get_student_simulation_instances(
+                                if start_dt:
+                                    if start_dt.tzinfo is None:
+                                        start_dt = start_dt.replace(tzinfo=timezone.utc)
+                                    if end_dt and end_dt.tzinfo is None:
+                                        end_dt = end_dt.replace(tzinfo=timezone.utc)
+                                    if end_dt:
+                                        time_delta = end_dt - start_dt
+                                        total_time_spent = max(0, int(time_delta.total_seconds()))
+                                        if abs(total_time_spent - (instance.total_time_spent or 0)) > 1:
+                                            instance.total_time_spent = total_time_spent
+                                            db.commit()
+                            except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)